docs/platform/release-notes/tizen-5-5-m1.md

```diff
@@ -0,0 +1,625 @@
+  - New Applications API set to get application's component type and category information has been added.
+  - New feature key for external storage has been added.
+  - New Bluetooth API set for handling Gatt client has been added.
+  - New AudioManager API set for the stream information has been added.
+  - New WebView API set to support more features has been added.
+  - New Sensor API set to cover all the sensors has been added.
+  - New Network API set to support Smart Traffic Control (STC) has been added.
+  - New Bluetooth API set to destroy Gatt server has been added.
+  - New SystemSettings API set for AccessibilityGrayscale and AccessibilityNegativeColor has been added.
+  - NUI API set for text related properties has been deprecated.
+  - New VoiceControlManager to support Voice Control Manager has been added.
+  - New InputMethod API set to handle input has been added.
+  - New DevicePolicyManager to support managing device policies has been added.
+  - New MediaContent API set for composer property has been added.
+  - New Bluetooth API set to add new Client property has been added.
+  - New Alarm API set for WeekFlag, Period, and ScheduledDate has been added.
+  - New Wi-Fi API set to provide accurate error information has been added.
+  - New WidgetControl API set for MainAppId, PackageId properties has been added.
+  - New DataControl API set for client appid and provider id has been added.
+  - New MediaTool API set for the VideoMediaFormat has been added.
+  - New AppControl API set to support sending the launch request asynchronously has been added.
+  - New InputMethod API set to update the input panel event has been added.
+  - New Application API set to add CurrentDeviceOrientation property for CoreUIApplication has been added.
+
+### Known Issues
+
+- Xamarin.Forms
+  - For more information on the list of limitations, see [here](https://developer.tizen.org/development/api-reference/.net-application/current-xamarin.forms-limitations).
+
+
+## Toolchain
+
+### New and changed features
+
+- ASan heap profile flags have been added to libsanitizer.
+- ASan __libc_siglongjmp/__libc_longjmp interceptors have been added.
+- Sanitizer feature has been improved.
+  - asan-runtime-env package has been added to pre-installation list of mic.
+  - ASan report description on stack traces has been improved.
+
+### Fixes
+
+- CVE patchset (CVE-2017-16997, CVE-2018-6485, CVE-2018-19591, CVE-2019-9169, CVE-2019-6488, and CVE-2019-7309) for glibc have been applied.
+- Bugfix patchset (#21081 and #22442) on glibc have been back-ported.
+
+
+## Experimental
+
+### New and changed features
+
+- nnfw: Neural Network Runtime (Experimental Release)
+  - Support for new architecture design named ‘neurun’ for CPU/GPU mixed inference has been added.
+  - Support for mixed acceleration for InceptionV3 and MobileNet model by manual planning has been added.
+  - Android NN API compatible (Supports major operations selected carefully.)
+
```

Comment:
Please, add nnstreamer feature. 

```bash
- nnstreamer: Neural Network Streamer
  - Added an efficient and flexible stream pipeline to neural networks.
  - Support a neural network model as a media filter and converter.
  - Support multiple neural network models in a single stream pipeline instance.
  - Support multiple sources and stream paths for neural network models.
  - Support RNN and CNN model architectures with Tensorflow, Tensorflow-lite.
  - Added neural network API based on NNStreamer to manage pipelines.
  - Added a plugin to support Linux IIO based sensors.
```

Reply:
Getting the consistency with other sentences. Please follow below form :

```
- nnstreamer: Neural Network Streamer
   - An efficient and flexible stream pipeline to neural networks has been added.
   - Support for a neural network model as a media filter and converter has been added.
   - Support for multiple neural network models in a single stream pipeline instance has been added.
   - Support for multiple sources and stream paths for neural network models has been added.
   - Support for RNN and CNN model architectures with Tensorflow, Tensorflow-lite has been added.
   - Neural network API based on NNStreamer to manage pipelines has been added.
   - A plugin to support Linux IIO based sensors has been added.
```

Reply:
> Getting the consistency with other sentences. Please follow below form :

@mijin85cho , Looks good to me. 
Acked-by: Geunsik Lim <geunsik.lim@samsung.com> 
Signed-off-by: Geunsik Lim <geunsik.lim@samsung.com>